packages/charts/src/hooks/use-distributed-chart-tooltip-settings.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  formatValue as valueFormatter,
+  getFromPath,
+  getKeysDifference,
+  ScaleSettings,
+  TooltipFormatter,
+  transformToPercent,
+} from '@keen.io/charts-utils';
+
+import { DataSelector } from '../types';
+import { getLabel } from '../components/distributed-chart-tooltip';
+
+const useDistributedChartTooltipSettings = (
```

Comment:
This `React` hook could be easily tested by using `@testing-library/react-hooks`. Could You please cover the code with all possible cases ?